mule-apikit-module/src/main/java/org/mule/module/apikit/validation/body/schema/v1/cache/JsonSchemaCacheLoader.java

```diff
@@ -85,6 +90,15 @@ private String resolveLocationIfNecessary(String path)
         return path;
     }
 
+    private String getEncodedPath(String path)
+    {
+        Set<Character> ignoredCharacters = new HashSet<>();
+        ignoredCharacters.add('/');
```

Comment:
We should consider \ too, as it may be used by windows users, and at some point we are checking the absolute path of the files. I don't know if URICoder is already ignoring that slash or not.